sql-plugin/src/main/scala/com/nvidia/spark/rapids/Plugin.scala

```diff
@@ -90,6 +90,14 @@ object RapidsPluginUtils extends Logging {
       logWarning("RAPIDS Accelerator is disabled, to enable GPU " +
         s"support set `${RapidsConf.SQL_ENABLED}` to true.")
     }
+
+    if (conf.isUdfCompilerEnabled) {
+      logWarning("RAPIDS UDF compiler is enabled, to disable it " +
```

Comment:
We might want to link to our FAQ https://nvidia.github.io/spark-rapids/docs/FAQ.html#automatic-translation-of-scala-udfs-to-apache-spark-operations . Without such context it's not clear for the user why she is warned about this setting. What makes it different from other switches we don't draw attention to? Looks like we consider it bleeding edge (experimental, unstable, evolving), and the wording for the warning should make it clear. E.g. 

> Experimental RAPIDS UDF compiler is enabled, in case of related failures disable it by ...

Reply:
Done!